mitodl/mitxonline-theme
# style: openedx theme logo updates

### What are the relevant tickets
[#3337](https://github.com/mitodl/hq/issues/3337)

### Description (What does it do?)
This PR updates logos in legacy edX theme for mitxonline.

### Screenshots (if appropriate):
Header and footer logo:
![openedx header and footer logo](https://github.com/mitodl/mitxonline-theme/assets/41048311/a3e90a50-ebad-4f2e-8495-77438d1c97f5)


Studio header logo:
<img width="1226" alt="studio logo" src="https://github.com/mitodl/mitxonline-theme/assets/41048311/020760ce-50d4-4595-9733-f13bf5c5fb47">

Favicon:
<img width="751" alt="openedx favicon" src="https://github.com/mitodl/mitxonline-theme/assets/41048311/f0b12d97-b064-47b4-bb03-4e53906c459a">

Emails (I couldn't test the actual emails but this is how it looks in the ace_templates):
![logo in email template](https://github.com/mitodl/mitxonline-theme/assets/41048311/305efece-a122-4259-b650-f29440a94543)

### How can this be tested?

1. Enable Theme.
   - Follow the instructions in the [Readme](https://github.com/mitodl/mitxonline-theme).

2. Add Settings in private.py:

```
LMS_BASE_URL = "http://localhost:18000"
LOGO_URL_PNG_FOR_EMAIL = f"{LMS_BASE_URL}/static/mitxonline-theme/images/logo.png"
LOGO_URL = f"{LMS_BASE_URL}/static/mitxonline-theme/images/logo.svg"
LOGO_TRADEMARK_URL = f"{LMS_BASE_URL}/static/mitxonline-theme/images/mit-ol-logo.svg"
BULK_EMAIL_SEND_USING_EDX_ACE = True
```

3. Verify Logos:
    - Check the updated logos in the legacy theme and the MFEs.
    - For any MFE, you will need to set the LOGO_URL and LOGO_TRADEMARK_URL in the .env.development.

4. Verify Logo in Ace Emails:
    - I couldn't test the logo with this [format](https://github.com/mitodl/mitxonline-theme/pull/43/commits/04dd7f3d4866992e49c1f754ace1e0bd97bf1eb8#diff-8837e91672139956ef9117136e4c27f5167cd666374543107691b8d33e3ef935R65) locally due to some settings issue, but it seems to work when mapping the actual logo URL in the src.
    - Reviewers can test this format locally if possible, as this is being picked for xPRO. Otherwise, we will test this directly on RC.
